app/Presenters/templates/Homepage/templateHowDoI.latte

```diff
@@ -0,0 +1,160 @@
+{block content}
+    <div class="container">
+        <img src="{$basePath}/img/logo.svg" id="logo">
+        <div class="center">
+            <p class="claim">Dev-stack solution based on Nette framework, which provides you with fully ready application skeleton
+                and environment <strong>to start developing a web application.</strong>
+            </p>
+            <pre>$ composer create-project instante/skeleton</pre>
+        </div>
+    </div>
+    <div class="blueline">
+        <div class="container center">
+            <div id="searchbox">
+                <span>How do I</span>
+                <input type="text" name="search_how" placeholder="Create" class="first">
+                <div class="autocomplete first">
+                    <div class="autocomplete__head">Found topics</div>
+                    <ul class="topics">
+                        <li>
+                            <a href="#">Create new application</a>
+                        </li>
+                        <li>
+                            <a href="#">Create Require.js module</a>
+                        </li>
+                        <li>
+                            <a href="#" class="more">... See all results >></a>
+                        </li>
+                    </ul>
+                    <div class="autocomplete__head">Not documented yet</div>
+                    <ul class="upvote__list">
+                        <li>
+                            <div class="upvote__item">
+                                <a href="#">
```

Comment:
@amcatko please don't do this as a link, only the upvote icon should be a link (so, add to some hover action and link only to the upvote icon)